assets/app/view/chat.rb

```diff
@@ -82,7 +82,32 @@ def render
       h('div#chat.half', props, children)
     end
 
+    private
+
+    QUOTED_BASE_URL = Regexp.quote(`window.location.origin || ''`)
```

Comment:
what happens if this is empty string?seems dangerous

Reply:
If the base URL is empty the third pattern reduces to `/game/`. This is not ideal but it neither causes a degenerate pattern nor is likely to produce many false positives.

The empty string only occurs in miniracer, and doesn't propagate to the actual rendered page because chat is loaded lazily post-load.